Library/Formula/hub.rb

```diff
@@ -17,7 +17,7 @@ class Hub < Formula
   depends_on "go" => :build
 
   def install
-    system "script/build"
+    system "script/build", "-o", "hub"
     bin.install "hub"
```

Comment:
If we can trust the group of the target directory in the Cellar, then the fewer lines, the better. If not, best to stage it in the temporary directory first.